GetandTake/Services/Concrete/ProductManager.cs

```diff
@@ -75,9 +77,19 @@ public async Task UpdateAsync(int productId, ProductDTO productDto)
             _repository.Update(product);
         }
     }
-    
+
     public void Delete(int productId)
     {
         _repository.Delete(product => product.ProductID == productId);
     }
+
+    public IEnumerable<ProductsDTO> GetByMaximumAmount(int maximumAmount)
+    {
+        if (maximumAmount is 0)
```

Comment:
You can simplify this expression in case the type of `maximumAmount` may change:
```suggestion
        if (maximumAmount == default)
```